sdk/keyvault/azure-security-keyvault-certificates/test/samples/certificate-get-certificates/certificate_get_certificates.cpp

```diff
@@ -157,8 +157,9 @@ KeyVaultCertificateWithPolicy CreateCertificate(
     // start the create process
     auto response = certificateClient.StartCreateCertificate(certificateName, options);
     // wait for complete to get the certificate
-    auto certificate = response.PollUntilDone(defaultWait).Value;
-
+    auto pollResponse = response.PollUntilDone(defaultWait).Value;
+    // get the certificate
+    auto certificate = certificateClient.GetCertificate(certificateName).Value;
```

Comment:
Call to GetCertificate() can depend on the `pollResponse` properties.  We can tell customers here how they can check if the LRO status is not error before getting the cert.